apps/challenges/models.py

```diff
@@ -444,3 +444,10 @@ class ChallengeEvaluationCluster(TimeStampedModel):
     class Meta:
         app_label = "challenges"
         db_table = "challenge_evaluation_cluster"
+
+
+def pre_save_post_receiver(sender, instance, *args, **kwargs):
```

Comment:
Can you please tell why are you doing this? @Kajol-Kumari

Reply:
It is used to call the `unique_slug_generator` function just before the saving of the `challengePhaseSplits`in order to generated the slug based on the title of the challenge.

Reply:
But it is already done when the challenge gets created [here](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/views.py#L1013) then what's the need of it here.

Reply:
Ah, i didn't noticed it. But yes, the `unique_slug_generator` i have used now gives preetier Url as the previously written one won't do much to an `id`